client/components/gsuite/gsuite-price/index.jsx

```diff
@@ -54,7 +54,7 @@ const GSuitePrice = ( { currencyCode, product } ) => {
 	return (
 		<div className="gsuite-price">
 			<h4 className="gsuite-price__monthly-price">
-				{ translate( '{{strong}}%(price)s{{/strong}} per user/month', {
+				{ translate( '{{strong}}%(price)s{{/strong}} /user /month', {
```

Comment:
I've updated this string to follow the same format than on the `Email Comparison` page.